resources/metadata/mavenStaticCodeChecks.yaml

```diff
@@ -0,0 +1,60 @@
+metadata:
+  name: mavenStaticCodeChecks
+  description: Execute static code checks using maven as is spobugs and pmd
```

Comment:
This looks weird. Also, I would consistently upper-case "Maven",  since it is an "Eigenname", and also "SpotBugs" and "PMD". But that's just personal preference.

Reply:
done